src/modules/uploader/uploader.ts

```diff
@@ -269,7 +269,10 @@ export class Uploader extends ViewComponent implements IUploader {
 					},
 					method: this.o.method || 'POST',
 					data: request,
-					url: this.o.url,
+					url:
+						typeof this.o.url === 'function'
```

Comment:
Jodit has helper `src/core/helpers/checker/is-function`
```js
if (isFunction(this.o.url))
```

Reply:
@xdan Thank you, I have changed the code to use the method.